bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
```

Comment:
Looks like this file was added by accident. Can you please remove it from this pull request?